Chell0/machtbewwust
# Add Responsive Navbar Section

## Description
This pull request adds a new responsive navbar section to the project.

### Changes
- Implemented a responsive navbar component
- Included navigation links to Home, Material, Themen, Blog, Databank, and Kalender sections
- Styled the navbar using Tailwind CSS
- Added hover and active states for the navigation links

## Screenshots
![Navbar Screenshot](https://github.com/Chell0/machtbewwust/assets/15849070/a749af65-caf0-4e5e-ac50-1d7786a97dde)

## How to Test
1. Pull this branch.
2. Run the development server using `bun dev`.
3. Verify the navbar displays correctly and is responsive on different screen sizes.
4. Check the functionality of the navigation links.
